utils.sh

```diff
@@ -53,6 +53,12 @@ function get_tag()
         echo "You are ahead of or behind a tagged release"
         VERSIONTAG="$COMMIT"
       fi
+      # save tag name + commit if availible
+      TAGNAME=$(git describe | sed -e 's/[\t ]*//')
```

Comment:
Should double quote chars (`"`) also be stripped out?
I took a look at [what is allowed in git tags](https://git-scm.com/docs/git-check-ref-format) and did not see anything preventing quotes. Even if it is a very unlikely situation, it could cause problems when the `version.js` is created above.

Example:
```
% TAGNAME=$(echo 'foo"bar')
% echo "var GUI_MONERO_VERSION = \"$TAGNAME\""
var GUI_MONERO_VERSION = "foo"bar"
```